recipe/meta.yaml

```diff
@@ -28,23 +31,26 @@ outputs:
         - setuptools
         - setuptools_scm
       run:
-        - cinnabar ~=0.4.0
-        - click <8.2
-        - gufe >=1.4.1,<1.5.0
+        - cinnabar ~=0.5.0
+        - click >=8.2.0
+        - gufe ~=1.5.0
         - kartograf ~=1.0
+        - konnektor~=0.2.0
         - lomap2 >=3.2.1
         - matplotlib-base
         - networkx
         - numpy
         - openfe-analysis ~=0.3.1
-        - openff-toolkit >=0.16,<0.17
-        - openmm !=8.1.0,>=8.0.0,<8.2
+        - openff-toolkit >=0.16.2,<0.17
+        - openff-units ==0.3.1
+        - openmm !=8.1.0,>=8.0.0,<8.3.0
         - openmmforcefields >=0.14.2,<0.15 # min pin b/c https://github.com/openmm/openmmforcefields/releases/tag/0.14.2 https://github.com/OpenFreeEnergy/openfe/discussions/1287
-        - openmmtools >=0.24.2,<0.25
+        - openmmtools >=0.25
         - packaging
+        - pint >=0.24.0
         - plugcli
         - pooch
-        - pydantic >=1.10.17
+        - pydantic >=2.0
```

Comment:
I don't think we need this? openfe imports pydantic.v1 which exists in 2.0 and 1.10.17